lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -2,6 +2,8 @@
 require 'fastlane/action'
 require 'fastlane/actions/github_api'
 require_relative 'github_helper'
+require_relative '../actions/bump_phc_version_action'
```

Comment:
hmm I'm not sure we should import an action from a helper... I know we are just using a constant there. Should we just move it to another file that can be accessed from both? Otherwise, I prefer moving the constant to the helper and have the action access it instead.

Reply:
This isn't an end of the world thing. I'd be fine if this is how it stayed 😇  But... might be nice to have a constants file for if/when other constants like this are added 🤷‍♂️